pkg/virt-controller/watch/export/export.go

```diff
@@ -74,15 +74,19 @@ const (
 	failedKeyFromObjectFmt = "failed to get key from object: %v, %v"
 	enqueuedForSyncFmt     = "enqueued %q for sync"
 
-	pvcNotFoundReason  = "pvcNotFound"
-	pvcBoundReason     = "pvcBound"
-	pvcPendingReason   = "pvcPending"
-	pvcInUseReason     = "pvcInUse"
-	unknownReason      = "unknown"
-	initializingReason = "initializing"
-	podPendingReason   = "podPending"
-	podReadyReason     = "podReady"
-	podCompletedReason = "podCompleted"
+	pvcNotFoundReason      = "pvcNotFound"
+	pvcBoundReason         = "pvcBound"
+	pvcPendingReason       = "pvcPending"
+	pvcInUseReason         = "pvcInUse"
+	unknownReason          = "unknown"
+	initializingReason     = "initializing"
+	podPendingReason       = "podPending"
+	podReadyReason         = "podReady"
+	podCompletedReason     = "podCompleted"
+	noVolumeSnapshotReason = "vmSnapshotNoVolumes"
+	notAllPVCsCreated      = "notAllPVCsCreated"
+	allPVCsReady           = "AllPVCsReady"
+	notAllPVCsReady        = "NotAllPVCsReady"
```

Comment:
Why sometimes start with caps, sometimes not?

Reply:
Don't know made them all upper case